src/BaGetter.Core/Configuration/BaGetterOptions.cs

```diff
@@ -43,6 +43,12 @@ public class BaGetterOptions
     /// </summary>
     public string Urls { get; set; }
 
+    /// <summary>
+    /// The maximum package size in GB.
+    /// Attempted uploads of packages larger than this will be rejected with an internal server error carrying one <see cref="System.IO.InvalidDataException"/>.
+    /// </summary>
+    public uint MaxPackageSizeGiB { get; set; } = 8;
```

Comment:
I used a bigger unit since this setting isn't interesting until you have huge packages, and when you have huge packages you won't really care about the specific number of bytes but the number of GB you can pump into this. This way, at least users with those use cases will indeed have a multiple of 2 as the maximum size, instead of just going "160000..." until they reach a giga because who actually wants to find out the specific number of bytes in 16GiB?

On the other hand I completely understand your sentiment, e.g. IIS uses bytes too for multiple settings there so I'd like another opinion on this.